.rubocop.yml

```diff
@@ -130,6 +130,15 @@ Style/FrozenStringLiteralComment:
     - 'actionpack/test/**/*.ruby'
     - 'activestorage/db/migrate/**/*.rb'
 
+Style/RedundantFreeze:
+  Enabled: true
+  Exclude:
```

Comment:
Why exclude these test files?

https://github.com/rails/rails/blob/master/actionpack/test/controller/test_case_test.rb
https://github.com/rails/rails/blob/master/activemodel/test/cases/type/string_test.rb
https://github.com/rails/rails/blob/master/activesupport/test/core_ext/string_ext_test.rb
https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/string/strip.rb
https://github.com/rails/rails/blob/master/railties/test/generators/actions_test.rb

Reply:
In my 5 second test, Rubocop didn't seem to complain about redundant `-"foo"`. If that's the case (and deliberate?), maybe we can consider switching to that for the first two. `string_ext_test.rb` seems more like it's missing a test for the _unfrozen_ case (though I'd really prefer to see all those wrapped in an `assert_deprecated` anyway).

Reply:
Yeah, I also confirmed about redundant `-"foo"` is no offence for the cop when I commented.
Since we already replaced all `String#dup` to `String#+@`, replacing redundant `String#freeze` to `String#-@` is fine to me.

Reply:
Thanks for the review. I have removed these files form exclude list and replaced `String#freeze` with `String#-@` where explicit string objects are necessary.